src/main/java/org/kiwiproject/champagne/core/manualdeployment/Release.java

```diff
@@ -0,0 +1,27 @@
+package org.kiwiproject.champagne.core.manualdeployment;
+
+import java.time.Instant;
+
+import javax.validation.constraints.NotBlank;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * The top-level container for all manual deployment tasks for a specific release.
+ */
+@Value
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Release {
```

Comment:
Would you want to change some of these to be `record` types, since this service is using Java 17? Doing it now will probably be less pain than later, and we'll learn where the pain points are. I am pretty sure Jackson now supports records, less sure about other things and where those problems might arise. But, if we think records are the way to go in the future, especially for simple types like this, then maybe give them a try. One downside is we lose the builder, unless Lombok has made a builder for records somehow?

Reply:
Let's do that as a future investigation.  I will add an issue.